pkg/cloud/aws.go

```diff
@@ -1,9 +1,44 @@
 package cloud
 
 import (
+	"bytes"
+	"embed"
+
+	appsv1 "k8s.io/api/apps/v1"
+	k8sYaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"k8s.io/klog/v2"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+//go:embed aws/assets/*
+var f embed.FS
+
 func GetAWSResources() []client.Object {
-	return []client.Object{}
+	resources := []struct {
+		obj   client.Object
+		asset string
+	}{
+		{&appsv1.Deployment{}, "aws/assets/deployment.yaml"},
```

Comment:
is the idea here that more resources will be added later?

Reply:
Correct, it is build with intention that it might need to deploy some SA or another Deployment potentially in the future.